### PR TITLE
remove github.com/kardianos/osext

### LINKED ISSUE #80127: github.com/kardianos/osext is deprecated
**What would you like to be added**: Use of github.com/kardianos/osext should be removed.

**Why is this needed**: It is deprecated and no longer necessary on Go 1.8+ (see [this comment, for example](https://github.com/kardianos/osext/issues/20#issuecomment-278110433)). Since all current versions of Kubernetes [require Go > 1.8](https://github.com/kubernetes/community/blob/master/contributors/devel/development.md#go), this requirement should not be an issue.

### PR DESCRIPTION
**Pull request by rusik69**
/kind cleanup

**What this PR does / why we need it**:
This PR removes deprecated dependency github.com/kardianos/osext
**Which issue(s) this PR fixes**:

Fixes #80127

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
NONE

```release-note
Remove deprecated github.com/kardianos/osext dependency
```

### CODE REVIEW

File: test/e2e_node/e2e_node_suite_test.go [hunk 1]
```diff
@@ -45,8 +45,6 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework/testfiles"
 	"k8s.io/kubernetes/test/e2e/generated"
 	"k8s.io/kubernetes/test/e2e_node/services"
-
```
Comment (BenTheElder) [inline test/e2e_node/e2e_node_suite_test.go]: Removing this blank line sepetating groups of imports requires the imports to be sorted, which fails gofmt.

We should leave this line.

**Comment by BenTheElder** [comment]
Please run `hack/update-vendor.sh`, removing this dependency should also remove it from vendor.

**Comment by cblecker** [comment]
@loqutus LGTM, but please squash your commits.

**Review by draveness — COMMENTED** [review]
/priority important-soon

**Comment by loqutus** [comment]
@cblecker squashed

**Comment by cblecker** [comment]
/lgtm
/approve

/unassign thockin
/assign @liggitt 
(Tim is on vacation)

**Comment by dims** [comment]
👍

**Pull request merged**